### PR TITLE
Cleanup of util/print.c

### PR DESCRIPTION
**Pull request by wargio**
# DO NOT SQUASH ME

**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

- Moved `rz_print_rowlog/rz_print_rowlog_done` to `rz_core_notify_begin/rz_core_notify_done` under rz_core
- Removed `rz_print_cursor_range` because does not exist.
- Removed `rz_print_c` because unused.
- Removed `rz_print_unset_flags` because unused.
- Moved `rz_print_hex_from_base2` under rz-ax since it is used only there.
- Removed `rz_print_hexdump_simple` because unused.
- Removed `rz_print_hexpairs` because unused.

### CODE REVIEW

**Comment by wargio** [comment]
Linked to https://github.com/rizinorg/rizin/issues/1392

File: librz/core/core.c [hunk 1]
```diff
@@ -24,6 +24,33 @@ static ut64 letter_divs[RZ_CORE_ASMQJMPS_LEN_LETTERS - 1] = {
 
 extern bool rz_core_is_project(RzCore *core, const char *name);
 
+RZ_API const char *rz_core_notify_begin(RzCore *core, const char *str) {
```
Comment (XVilka) [inline librz/core/core.c]: This function certainly needs a description on what exactly "notify" means.

Comment (XVilka) [reply librz/core/core.c]: Please also check `str` for NULL and add `RZ_NONNULL`

File: librz/core/core.c [hunk 2]
```diff
@@ -24,6 +24,33 @@ static ut64 letter_divs[RZ_CORE_ASMQJMPS_LEN_LETTERS - 1] = {
...
+	}
+	if (use_color) {
+		rz_cons_printf("[ ] " Color_YELLOW "%s\r[" Color_RESET, str);
+	} else {
+		rz_cons_printf("[ ] %s\r[", str);
+	}
+	return str;
+}
+
+RZ_API void rz_core_notify_done(RzCore *core, const char *str) {
```
Comment (XVilka) [inline librz/core/core.c]: Please also check `str` for NULL and add `RZ_NONNULL`

File: librz/util/print.c [hunk 3]
```diff
@@ -1195,23 +1180,6 @@ RZ_API void rz_print_raw(RzPrint *p, ut64 addr, const ut8 *buf, int len) {
...
-		if (!(i % inc)) {
-			p->cb_printf("\n");
-		}
-	}
-	p->cb_printf(" };\n");
-}
-
 // HACK :D
 static RzPrint staticp = {
 	.cb_printf = libc_printf
```
Comment (XVilka) [inline librz/util/print.c:1185]: Is this struct/member still required? Maybe remove them as well?

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/util/print.c (hunk 3 above)
Comment (wargio) [reply librz/util/print.c:1185]: yes, it is for now. when i will finish with all prs, it will be gone

**Review by XVilka — CHANGES_REQUESTED** [review]
LGTM but one test requires to be fixed first:
```
[XX] db/analysis/dalvik Dalvik resolve virtual methods called via invoke-virtual

RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'e scr.prompt=false ; aaa ; axt @ sym.Lcom_ice_tar_TarEntry_.getUserName__Ljava_lang_String' apk://bins/dex/org.radare.radare2installer.apk

-- stdout

--- expected

+++ actual

@@ -1,3 +1,14 @@

 entry0 0xc8ba [CALL] invoke-virtual {v1}, Lcom/ice/tar/TarEntry;->getUserName()Ljava/lang/String;

 entry0 0xc9e6 [CALL] invoke-virtual {v1}, Lcom/ice/tar/TarEntry;->getUserName()Ljava/lang/String;

 method.public.Lcom_ice_tar_TarEntry_.toString__Ljava_lang_String 0xf04a [CALL] invoke-virtual {v4}, Lcom/ice/tar/TarEntry;->getUserName()Ljava/lang/String;

+WARNING: rz_core_notify_done: assertion 'core && message' failed (line 57)

+WARNING: rz_core_notify_done: assertion 'core && message' failed (line 57)

+WARNING: rz_core_notify_done: assertion 'core && message' failed (line 57)

+WARNING: rz_core_notify_done: assertion 'core && message' failed (line 57)

+WARNING: rz_core_notify_done: assertion 'core && message' failed (line 57)

+WARNING: rz_core_notify_done: assertion 'core && message' failed (line 57)

+WARNING: rz_core_notify_done: assertion 'core && message' failed (line 57)

+WARNING: rz_core_notify_done: assertion 'core && message' failed (line 57)

+WARNING: rz_core_notify_done: assertion 'core && message' failed (line 57)

+WARNING: rz_core_notify_done: assertion 'core && message' failed (line 57)

+WARNING: rz_core_notify_done: assertion 'core && message' failed (line 57)

-- stderr

WARNING: No calling convention defined for this file, analysis may be inaccurate.
```
https://app.travis-ci.com/github/rizinorg/rizin/jobs/564808470#L3682

**Pull request merged**